libcontainer/container_linux_test.go

```diff
@@ -297,19 +297,24 @@ func TestGetContainerState(t *testing.T) {
 		}
 	}
 	for _, ns := range container.config.Namespaces {
-		path := state.NamespacePaths[ns.Type]
+		path, ok := state.NamespacePaths[ns.Type]
+		if !configs.IsNamespaceSupported(ns.Type) {
+			if ok {
+				t.Fatalf("namespace %s is not supported", ns.Type)
+			}
+			continue
+		}
 		if path == "" {
 			t.Fatalf("expected non nil namespace path for %s", ns.Type)
 		}
+
 		if ns.Type == configs.NEWNET {
 			if path != expectedNetworkPath {
 				t.Fatalf("expected path %q but received %q", expectedNetworkPath, path)
 			}
 		} else {
-			file := ""
+			var file string
 			switch ns.Type {
-			case configs.NEWNET:
-				file = "net"
 			case configs.NEWNS:
 				file = "mnt"
 			case configs.NEWPID:
```

Comment:
All this can probably be replaced with a call to `ns.GetPath()` (which was added later than this test).